CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ Initial release of phac-nml/mikrokondo. Mikrokondo currently supports: read trim
 
 ### `Fixed`
 
+- Matched ECTyper and SISTR parameters to what is set in the current IRIDA addressing issue [#58](https://github.com/phac-nml/mikrokondo/issues/58)
```

Comment:
Once https://github.com/phac-nml/mikrokondo/pull/66 gets merged in, can you merge `dev` into this branch and move this `Fixed` bullet point to the **Unreleased** section.

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/68/commits/9beb0f111c0d4495ac98e3e0336116b954e27257